pkg/dao/task_info.go

```diff
@@ -116,12 +116,12 @@ func (t taskInfoDaoImpl) List(
 }
 
 func (t taskInfoDaoImpl) Cleanup(ctx context.Context) error {
-	// Delete all completed or failed introspection tasks that are older than 10 days
-	// Delete all finished Repo delete tasks that are older 10 days
+	// Delete all completed or failed specified tasks that are older than 10 days
+	// Delete all finished delete tasks that are older 10 days
 	q := "delete from tasks where " +
-		"(type = '%v' and (status = 'completed' or status = 'failed') and finished_at < (current_date - interval '10' day)) OR" +
-		"(type = '%v' and status = 'completed' and  finished_at < (current_date - interval '10' day))"
-	q = fmt.Sprintf(q, config.IntrospectTask, config.DeleteRepositorySnapshotsTask)
+		"(type in (%v) and (status = 'completed' or status = 'failed') and finished_at < (current_date - interval '10' day)) OR" +
+		"(type in (%v) and status = 'completed' and  finished_at < (current_date - interval '10' day))"
```

Comment:
can we adjust this 10 days to 20 days?  10 was fine for something like introspection failures, but for other failures, i worry 10 isn't enough retention to be able to investigate problems adequately.

Reply:
sorry, not this one, the one above it, with 'or status = failed'

Reply:
updated!